docs/src/components/DesignTokensList/components/DesignTokenValue.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { Truncate, Tooltip } from "@kiwicom/orbit-components";
+import { Check } from "@kiwicom/orbit-components/icons";
+
+import CopyIcon from "../../../images/icons/CopyIcon.svg";
+import useCopyToClipboard from "../../../hooks/useCopyToClipboard";
+
+const StyledCopyButton = styled.button<{ $isCopied: boolean }>`
+  ${({ $isCopied }) => css`
+    height: 24px;
+    width: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    opacity: ${$isCopied ? "1" : "0"};
+  `}
+`;
+
+const StyledTokenValue = styled.span`
+  display: inline-flex;
+  align-items: center;
+
+  :hover {
+    ${StyledCopyButton} {
+      opacity: 1;
+    }
+  }
+`;
+
+const StyledStrikeThrough = styled.span<{ $hasStrikeThrough?: boolean }>`
+  ${({ $hasStrikeThrough }) => css`
+    ${$hasStrikeThrough && "text-decoration: line-through"};
+    opacity: 0.5;
+  `}
+`;
+
+const DesignTokenValue = ({
+  value,
+  deprecated,
+  hasStrikeThrough,
+  replacement,
+  showCopyButton,
+  width = "200px",
+}: {
+  value: string | number;
+  replacement?: string | null;
+  deprecated?: boolean;
+  hasStrikeThrough?: boolean;
+  showCopyButton?: boolean;
+  width?: string;
+}) => {
+  const [isCopied, copy] = useCopyToClipboard(2000);
+
+  const wrapValue = () => <Truncate maxWidth={width}>{value}</Truncate>;
+
+  const renderValue = () => {
+    if (deprecated) {
+      return (
+        <Tooltip
```

Comment:
Sometimes, this Tooltip is not wide enough for the message. Can you please adjust?

<img width="595" alt="image" src="https://github.com/kiwicom/orbit/assets/6265045/812c61de-2278-4ac0-8c85-fa07d7c09c70">